### PR TITLE
reword open_port dialogue

### PR DESCRIPTION
**Pull request by hboetes**
I asked bard to reword this sentence. It looks more natural indeed.

### CODE REVIEW

File: pynicotine/gtkgui/ui/dialogs/fastconfigure.ui [hunk 1]
```diff
@@ -253,7 +253,7 @@
             <property name="visible">True</property>
             <child>
               <object class="GtkLabel">
-                <property name="label" translatable="yes">Nicotine+ uses peer-to-peer networking to connect to other users. In order to allow users to connect to you without trouble, an open listening port is crucial.</property>
+                <property name="label" translatable="yes">Nicotine+ uses a peer-to-peer network to connect to other users. To make sure that other users can connect to you without problems, it is important to have an open listening port.</property>
```
[the remark this answers is not in the document]
Comment (hboetes) [reply pynicotine/gtkgui/ui/dialogs/fastconfigure.ui:256]: I like that idea. I think this is even clearer:

To make connections with other Soulseek peers, a listening port on your router has to be forwarded to your computer.

[the remark this answers is not in the document]
Comment (mathiascode) [reply pynicotine/gtkgui/ui/dialogs/fastconfigure.ui]: Could this be simplified further? `To make connections with other Soulseek peers` -> `To connect to other Soulseek peers`

The first thing that comes to my mind when reading `make connections` is getting to know someone, or making friends.

[the remark this answers is not in the document]
Comment (hboetes) [reply pynicotine/gtkgui/ui/dialogs/fastconfigure.ui]: That's what slsk is all about. ;-)

I changed the phrase as per your suggestion.

[the remark this answers is not in the document]
Comment (slook) [reply pynicotine/gtkgui/ui/dialogs/fastconfigure.ui]: Can we have `To connect with other Soulseek peers`?

"with" indicates bi-directionality and avoids repetition of "to".

[the remark this answers is not in the document]
Comment (hboetes) [reply pynicotine/gtkgui/ui/dialogs/fastconfigure.ui]: Good one, done.

**Pull request merged**